src/client.ts

```diff
@@ -772,6 +816,48 @@ export class Client {
     bindingKeys.map((bk) => partitions.push(`${streamName}-${bk}`))
     return { partitions, streamBindingKeys: bindingKeys }
   }
+  private incrRefCount() {
+    return ++this.refCount
```

Comment:
the return value is used? otherwise i'll remove the return

Reply:
ok, I'll remove the return statement